codespell_lib/data/dictionary.txt

```diff
@@ -1184,6 +1184,7 @@ accumualtion->accumulation
 accumualtive->accumulative
 accumualtor->accumulator
 accumualtors->accumulators
+accumuate->accumulate
```

Comment:
It would be great if you could add all verbal forms:
```suggestion
accumuate->accumulate
accumuated->accumulated
accumuates->accumulates
accumuating->accumulating
```